src/store/modules/rootReducer.js

```diff
@@ -0,0 +1,4 @@
+import { combineReducers } from 'redux';
+import currentUser from './currentUser/reducer';
```

Comment:
Change the name to `userReducer`. It makes more sense.

Reply:
it's done